lib/model-builder.js

```diff
@@ -257,6 +257,12 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
       deprecated('Property names containing a dot are not supported. ' +
         'Model: ' + className + ', property: ' + p);
     }
+    
+    // Warn if property name is 'constructor'
+    if (p === "constructor") {
+      deprecated('Property name should not be constructor in Model: ' +
```

Comment:
Could you please put "constructor" in quotes too? `...should not be "constructor" in Model:...`